helper.go

```diff
@@ -20,6 +20,7 @@ type helper interface {
 	init(*sql.DB) error
 	disableReferentialIntegrity(*sql.DB, loadFunction) error
 	paramType() int
+	setParamType(int)
```

Comment:
there is `baseHelper` struct, which is inherited by all of the drivers and it is good place to have:
* setCustomParamType: used by `WithPlaceholder`
* getDefaultParamType: the default param type e.g dollar for postgres
* getParamType: conditional logic `override or getDefaultParamType()`

Reply:
done

Reply:
> getDefaultParamType: the default param type e.g dollar for postgres
getParamType: conditional logic override or getDefaultParamType()

this part can't be implemented in baseHelper, because baseHelper don't know anything about struct where it used as composition

Reply:
sorry i wasn't clear enough:
* all 3 methods are defined in an interface
* baseHelper define `setCustomParamType` and `getDefaultParamType`, so driver specific helper does not have to this
* each driver has a `getDefaultParamType` implementation 

Assuming those points above please inform me, if there is some gap in understanding how it can work